examples/java-agent/pom.xml

```diff
@@ -0,0 +1,100 @@
+<project>
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>example</groupId>
+  <artifactId>helloworld</artifactId>
+  <version>1</version>
+
+  <properties>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <jib-maven-plugin.version>0.9.11</jib-maven-plugin.version>
+    <maven-compiler-plugin.version>3.8.0</maven-compiler-plugin.version>
+    <download-maven-plugin.version>1.4.1</download-maven-plugin.version>
+
+    <!-- agents will be extracted relative to this build location -->
+    <agent-extraction-root>${project.build.directory}/agents</agent-extraction-root>
+    
+    <!-- location of Stackdriver Debugger Java Agent https://cloud.google.com/debugger/docs/setup/java -->
+    <stackdriver-debugger-agent-url>https://storage.googleapis.com/cloud-debugger/compute-java/debian-wheezy/cdbg_java_agent_gce.tar.gz</stackdriver-debugger-agent-url>
```

Comment:
I wonder if it makes sense to pin down to a specific version or just embed the binary for build reproducibility.

Reply:
They don't do versioned releases, from what I could find.